src/wp-includes/media.php

```diff
@@ -5298,12 +5298,17 @@ function get_post_galleries( $post, $html = true ) {
 			// New Gallery block format as an array.
 			if ( $has_inner_blocks ) {
 				$attrs = wp_list_pluck( $block['innerBlocks'], 'attrs' );
-				$ids   = wp_list_pluck( $attrs, 'id' );
 
-				foreach ( $ids as $id ) {
-					$url = wp_get_attachment_url( $id );
+				$ids = array();
+				foreach ( $attrs as $attr ) {
+					if ( isset( $attr['id'] ) ) {
```

Comment:
```suggestion
					if ( ! isset( $attr['id'] ) ) {
```

It only want to process attributes that have an ID.